### DISCUSSION TITLE
Normal, Horizontal, and Global Irradiance in Solar Data

### DISCUSSION THREAD

**Discussion by jahanbani** [blaylockbk/Herbie#119] [Q&A] [answered]
Hello,
I am using the HRRR dataset to measure the solar energy potential at each lat/lon using sub-hourly data. I use `subh` and `fx=0` (when I open the file, I read the 0-, 15-, 30-, and 45-minute data).
The only value related to solar seems to be `Downward short-wave radiation flux`. 

1- To calculate the PV potential, I need the normal and horizontal irradiance. Does HRRR contain this information? Or is the Downward short-wave radiation flux is global irradiance value (I am assuming the global irradiance value considers normal and horizontal irradiance)?

2- What is the timezone in the HRRR dataset? I am getting irradiance at 10pm on January 1st 😁.

**Comment by blaylockbk** [comment, marked as answer]
Could you re-post these questions on the GSL Regional Model Forum? Thanks!
https://github.com/NOAA-GSL/GSL-Regional-Model-Forum/discussions

**Reply by jahanbani** [reply]
Thank you very much. Will do 😄
